Source/FlowEditor/Private/Graph/FlowGraphSchema.cpp

```diff
@@ -72,17 +74,48 @@ void UFlowGraphSchema::GetGraphContextActions(FGraphContextMenuBuilder& ContextM
 
 void UFlowGraphSchema::CreateDefaultNodesForGraph(UEdGraph& Graph) const
 {
+	const UFlowAsset* AssetClassDefaults = GetAssetClassDefaults(&Graph);
+
+	static const FVector2D NodeOffsetIncrement = FVector2D(0, 128);
```

Comment:
_Offsetting by 128 puts each of the CustomInputs nicely on one of the Graph editor "thick" lines in its grid, which looks nice and tidy.  It also gives a decent (but not overly so) amount of room between each._